src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+  --dark-blue: rgba(61, 78, 97, 1);
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 350;
+}
```

Comment:
only `box-sizing` is allowed be in `*` selector. Move variable into `:root`, `font-family` into `html` 
`font-weigh` has incorrect value